tfb.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+docker run --network=host -v //var/run/docker.sock:/var/run/docker.sock --mount type=bind,source=c:/Development/FrameworkBenchmarks,target=/FrameworkBenchmarks tfb "$@"
```

Comment:
Is `--net=host` really required here? Also there's two `/` in `//var`, plus the whole `--mount` option can be shortened to `` -v `pwd`:/FrameworkBenchmarks``